exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -455,6 +455,8 @@ drill.exec.options: {
     exec.hashjoin.num_partitions: 32,
     exec.hashjoin.num_rows_in_batch: 1024,
     exec.hashjoin.max_batches_in_memory: 0,
+    exec.hashjoin.enable.runtime_filter: true,
```

Comment:
I would suggest to keep the default value as false until the feature goes through performance and functional testing.

Reply:
agree